content/api/control-api.textile

```diff
@@ -1,9 +1,10 @@
 ---
-title: API Reference
+title: Control API Reference
 meta_description: "The Control API is a REST API that enables you to manage your Ably account programmatically. This is the Control API Reference guide."
 meta_keywords: "Ably, Control API, account management, control, REST API, OpenAPI Specification, OAS3."
-section: control-api
-index: 10
+section: api
+index: 200
 redoc: true
 spec_url: "https://raw.githubusercontent.com/ably/open-specs/main/definitions/control-v1.yaml"
+redirect_from: /control-api/api-reference
```

Comment:
The redirect doesn't work when I test this locally.

Reply:
That's odd. They work fine in the nanoc view: both the deep links and the redirect. However, under the web view neither work (locally). Not sure why not.

Reply:
I've looked into this with Ken's help. There's some website code that needs to be changed to support the new file location. We can flag that up when we do the docs bump containing this change. Ken will make the required change to the website code as part of that docs bump PR.